### PR TITLE
libp2p v0.41.0

### PR DESCRIPTION
**Pull request by wngr**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-broadcast"
-version = "0.6.0"
+version = "0.6.1"
```
Comment (dvc94ch) [inline Cargo.toml]: needs to be a minor bump

**Comment by dvc94ch** [comment]
Can you copy the ci config from libp2p-bitswap or ipfs-embed?

**Comment by wngr** [comment]
sure

**Review by dvc94ch — APPROVED** [review]
approved if ci is green

**Comment by wngr** [comment]
@dvc94ch can you publish and/or give me access rights please?

**Comment by dvc94ch** [comment]
you should have access now

**Comment by wngr** [comment]
Thanks! Released 0.7.0

**Pull request merged**